src/main/java/com/codejoust/main/model/User.java

```diff
@@ -37,8 +45,23 @@
      */
     private String sessionId;
 
+    // This column holds the primary key of the account
+    @ManyToOne(fetch = FetchType.EAGER)
+    @JoinColumn(name = "account_table_id")
+    private Account account;
+
     // This column holds the primary key of the room (not the roomId variable)
     @ManyToOne(fetch = FetchType.EAGER)
     @JoinColumn(name = "room_table_id")
     private Room room;
+
+    // This column holds the primary key of the game report
+    @ManyToOne(fetch = FetchType.EAGER)
+    @JoinColumn(name = "game_report_table_id")
+    private GameReport gameReport;
+
+    // Thie list holds the submission group associated with the current room
+    @OneToMany(mappedBy = "user", fetch = FetchType.EAGER)
+    @Setter(AccessLevel.PRIVATE)
+    private List<SubmissionGroupReport> submissionGroupReports = new ArrayList<>();
```

Comment:
Is my thinking correct here (just wanting to confirm)? From understanding, when a GameReport is fetched, you'll have the list of users just fine. Then, to see what each player submitted, you would loop through this field and find the one index whose SubmissionGroupReport's gameId belongs to that GameReport?

Reply:
If so, maybe we could manually implement a getter method along the lines of getSubmissions (either here or in GameReport) that would filter through and find the correct report. 

Along a similar line, it could be useful when mapping the GameReport to a GameReportDto, to automatically do that (meaning GameReportDto's user DTO would simply hold a single SubmissionGroupReport)

Reply:
I think all this is correct - the implementation and details / whatnot will be on the next PR, but if I understand correctly that's what I'm thinking too 👍